packages/usdk/cli.js

```diff
@@ -1850,30 +1821,57 @@ export const main = async () => {
           await disable(args);
         });
       }); */
-    program
-      .command('voice')
-      .description(
-        'Manage agent voices',
-      )
-      .argument(
-        `[subcommand]`,
-        `What voice action to perform; one of [${voiceSubCommands.map(cmd => cmd.name).join(', ')}]`,
-      )
-      .argument(
-        `[args...]`,
-        `Arguments to pass to the subcommand`,
-      )
-      .action(async (subcommand = '', args = [], opts = {}) => {
-        await handleError(async () => {
-          commandExecuted = true;
-          args = {
-            _: [subcommand, args],
-            ...opts,
-          };
-          await voice(args);
-        });
-      })
-      .addHelpText('after', `\nSubcommands:\n${voiceSubCommands.map(cmd => `  ${cmd.name}\t${cmd.description}\n\t\t${cmd.usage}`).join('\n')}`);
+    const voiceSubCommands = [
```

Comment:
This looks like an incorrect merge. It might be undoing previous work.

Reply:
Ah shoot. Crazy. `git` seems to be glitching on me.